zerver/tests/test_custom_profile_data.py

```diff
@@ -556,7 +543,7 @@ def test_update(self) -> None:
         field.refresh_from_db()
         self.assertEqual(field.hint, "")
 
-    def test_display_in_profile_summary(self) -> None:
+    def test_update_display_in_profile_summary(self) -> None:
```

Comment:
I guess this should be in the previous commit which adds the test.

Reply:
my bad, fixed!